api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -5,12 +5,17 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified-with-null-constraint-and-default">
+    <preConditions onFail="MARK_RAN" > 
```

Comment:
No, we can't have this. The entire point of Liquibase is that it's idempotent, and if it's at a state, it got there by playing everything before it successfully.

Can we just restore the database from a backup and run a migration from there?

Reply:
This was not the ideal solution we wanted to do however I had to do this just because i cannot run ALTER commands on test DB and we deleted the entry in databasechangelog for changeset

Reply:
That's kinda what I figured.

Reply:
We will definitely need a way to obtain such access for the oncall engineer when needed.